js/join.js

```diff
@@ -410,3 +413,95 @@ submit.addEventListener('click', async () => {
 copyBtn.addEventListener('click', () => {
   navigator.clipboard.writeText(`Real Dev Squad Verification Link: ${url}`);
 });
+
+document.querySelector('#introduction').addEventListener('mouse', (event) => {
```

Comment:
why are you using mouse event??

Reply:
Done, video is attached in the description. Could you please look into it. Thank you